packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -36,6 +36,7 @@ export 'package:flutter/gestures.dart' show
   ScaleEndDetails,
   TapDownDetails,
   TapUpDetails,
+  ForcePressDetails,
```

Comment:
Add the other ones while you're here if there are

Reply:
There aren't actually.